.github/workflows/e2e.generic-bazel.tag.main.default.slsa2.yml

```diff
@@ -0,0 +1,118 @@
+name: Generic (Bazel) tag main default SLSA2
+
+on:
+  schedule:
+    - cron: "0 2 * * *"
+  workflow_dispatch:
+  push:
+    tags:
+      - "*" # triggers only if push new tag version, like `0.8.4` or else
+
+env:
+  GH_TOKEN: ${{ secrets.E2E_GO_TOKEN }}
+  ISSUE_REPOSITORY: slsa-framework/slsa-github-generator
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'schedule' || github.event_name == 'workflow_dispatch'
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 #v2.4.0
+      - id: create
+        # Note: we use v21.x.y
```

Comment:
If I understand the `README.md` correctly, after this lands we need to manually create a 21.0.0 release, right?

Reply:
You're correct. Set the description to something like `manual setup for <filename>`. Upon new triggers (daily thru schedule), this script https://github.com/slsa-framework/example-package/blob/main/.github/workflows/scripts/e2e-create-release.sh#L16 looks for the name of the file in the description to identify the release version to bump.

We have this issue to address the manual step https://github.com/slsa-framework/example-package/issues/34. Basically we're going to add an env variable. We'll be able to also have pre-submit of some sanity check that no workflows share the same version range.